easybuild/easyconfigs/o/OpenKIM/make-kim-model-list.py

```diff
@@ -0,0 +1,32 @@
+from urllib.request import urlopen
+import json
+import time
+
+query = b'&fields={"kimcode":1,"kim-api-version":1,"driver":1}&database=obj&query={"type":"mo"}'
+#query = b'&database=obj&query={"type":"mo"}'
+url = 'https://query.openkim.org/api'
+timeout = 60        
+# We should maybe do some error handling here?
+url_fd = urlopen(url, data=query, timeout=timeout)
```

Comment:
Maybe take a look at https://github.com/hpcugent/easybuild-framework/blob/master/easybuild/tools/filetools.py#L408 for inspiration w.r.t. error handling.

Also, it's better to use `urllib2` which is aware of proxies.

It's OK to include as-is though, up to you if you want to spend more time on this.